src/components/Blog/FeaturedPost/index.tsx

```diff
@@ -6,50 +6,61 @@ import blogCss from '../styles.module.css'
 import { formatBlogDate } from '@/components/Blog/utils/formatBlogDate'
 import { calculateReadingTimeInMin } from '@/components/Blog/utils/calculateReadingTime'
 import { type BlogPostEntry } from '@/components/Blog/Post'
-import { isAsset } from '@/lib/typeGuards'
+import { isAsset, isEntryTypeTag } from '@/lib/typeGuards'
 import CategoryIcon from '@/public/images/Blog/category-icon.svg'
 import { AppRoutes } from '@/config/routes'
+import some from 'lodash/some'
+import { PRESS_RELEASE_TAG } from '@/components/Blog/BlogHome'
 
 const FeaturedPost = (props: BlogPostEntry) => {
   const { slug, coverImage, category, date, title, excerpt, tags, content } = props.fields
 
+  const isPressRelease = some(tags?.filter(isEntryTypeTag), (item) => item.fields.name === PRESS_RELEASE_TAG)
```

Comment:
I'd save the import and use the native `some` function:

```suggestion
  const isPressRelease = tags?.filter(isEntryTypeTag)?.some((item) => item.fields.name === PRESS_RELEASE_TAG)
```